css/properties/position-area.json

```diff
@@ -40,7 +39,7 @@
             "webview_ios": "mirror"
           },
           "status": {
-            "experimental": true,
+            "experimental": false,
```

Comment:
Although the data guidelines [don't specify this](https://github.com/mdn/browser-compat-data/blob/a244bf0b3ffe1f897c4e3bafd0e62ce211f694f0/docs/data-guidelines/index.md?plain=1#L122-L133), we should probably consider a partial implementation as no support for the experimental setting.